curvefs/src/metaserver/partition.cpp

```diff
@@ -220,10 +220,6 @@ MetaStatusCode Partition::GetInode(uint32_t fsId, uint64_t inodeId,
         return MetaStatusCode::PARTITION_ID_MISSMATCH;
     }
 
-    if (GetStatus() == PartitionStatus::DELETING) {
```

Comment:
why remove this check?

Reply:
a deleting partition can read, not write